Buildings/Examples/VAVReheat/Validation/TraceSubstance.mo

```diff
@@ -0,0 +1,13 @@
+within Buildings.Examples.VAVReheat.Validation;
+model TraceSubstance
+  "This validates the ability to simulate trace substances in the air"
+  extends ASHRAE2006(
+    MediumA(extraPropertiesNames={"CO2"}));
+  annotation (experiment(
+      StartTime=3500000,
+      StopTime=4650000,
+      Interval=900,
+      Tolerance=1e-06),
+      __Dymola_Commands(file="modelica://Buildings/Resources/Scripts/Dymola/Examples/VAVReheat/Validation/TraceSubstance.mos"
+        "Simulate and plot"));
```

Comment:
Add a brief `info` and `revision` section. All models should have this.

Reply:
Ah yes, I'll do this, sorry I forgot.

Reply:
Added info and revisions for each new model in https://github.com/lbl-srg/modelica-buildings/pull/2486/commits/96a59b638856cb64c343a3536451699ca5450a6b.